packages/devtools-modules/client/framework/menu.js

```diff
@@ -115,9 +115,12 @@ Menu.prototype._createMenuItems = function (parent) {
       let menupopup = doc.createElement("menupopup");
       item.submenu._createMenuItems(menupopup);
 
+      let menuitem = doc.createElement("menuitem");
```

Comment:
I think it's hard.  Maybe we can add more CSS workarounds here in order to display context menu properly. However, it would be very ugly in my opinion and need to take more effort to get workaround work.